src/docs/JobPlanner/references/JobPlannerServiceReference.adoc

```diff
@@ -299,4 +299,145 @@ In such case, an *Association failed* notification will be triggered to users su
+For example, if the user belongs to 2 groups and each group has a different type of access, the system will choose the access with the highest priority for the user.
+For an authorization assigned to a user, the value is 0, since it will be ignored when calculating the resulting access type.
+
+To create a new authorization, the admin just needs to click on the image:../../images/Add-authorisation-button.png[] button.
+Then the view of creating an authorization (as shown in the figure below) will be popped up.
+The admin needs to fill in the authorization attributes (as presented above), then click the "Add authorization" button.
+The figure below shows an example of an authorization which gives the user named `Olivia` `write` access to association `1`.
+
+image::../../images/Create-association-authorisation.png[align=center]
+
+Once the admin has created an authorization, it will appear in the authorizations list in the authorization view, as shown in the figure below.
+
+image::../../images/List-of-authorisations.png[align=center]
+
+Once the first authorization has been created, the admin will see a "shield" icon next to the association to which the authorization was created.
+The figure below shows an example.
+
+image::../../images/Admin-association-shield.png[align=center]
+
+Moreover, when the user, who is benefiting from the authorization, log in to his/her account, he/she will be able to visualize an icon indicating the rights he/she possess over the association(s).
+These icons are the following:
+
+image::../../images/Association-authorrizations-icons.png[align=center]
+
+* The first icon indicates that the user has `read` rights over the association(s).
+* The second icon indicates that the user has `write` rights over the association(s).
+* The third icon indicates that the user has `admin` rights over the association(s).
+
+The figure below shows how the user visualize the authorization icon next to the association.
+
+image::../../images/User-association-icon.png[align=center]
+
+To delete an authorization, the admin simply needs to select one from the list and the delete button will be activated.
+It is the button with a "bin" symbol next to the add a new authorization button "+".
+The figure below shows an example.
+
+image::../../images/Delete-association-authorisation.png[align=center]
+
+Once the admin delete the authorization, it will be removed from the list as shown in the figure below.
+
+image::../../images/Association-authorisations-view.png[align=center]
+
+To update an authorization, the admin needs to click on the drop-down list of the access type or the priority level and select a new value.
+The figure below shows an example.
+
+image::../../images/Update-association-authorisation.png[align=center]
+
+Once the selection is made the authorization will be updated as shown in the figure below.
+
+image::../../images/Updated-association-authorisation.png[align=center]
+
```

Comment:
please add a sub chapter "Manage authorizations for all associations owned by the user"